apps/api/src/billing/repositories/usage/usage.repository.ts

```diff
@@ -0,0 +1,111 @@
+import { QueryTypes } from "sequelize";
+import { singleton } from "tsyringe";
+
+import { chainDb } from "@src/db/dbConnection";
+
+interface BillingUsageRawResult {
+  Date: string;
+  "Active Leases": number;
+  "Daily AKT Spent": number;
+  "Total AKT Spent": number;
+  "Daily USDC Spent": number;
+  "Total USDC Spent": number;
+  "Daily USD Spent": number;
+  "Total USD Spent": number;
```

Comment:
Non-blocking: this notation looks weird as internal query. I'd suggest using smth more usual like snake/camel case.

Reply:
+1
Used camelCase